.appveyor.yml

```diff
@@ -1,6 +1,8 @@
-﻿version: 2.3.2.{build}
+version: 2.3.2.{build}
```

Comment:
ugh this line-ending change is back. I will try to revert it.

Reply:
fixed it in https://github.com/microsoft/LightGBM/pull/2965/commits/c53cbcad8beeafc6e0228579c048da2f7c668a08

![image](https://user-images.githubusercontent.com/7608904/81464190-db596180-9184-11ea-9b8e-f02538500a44.png)